hack/run-e2e-test

```diff
@@ -78,7 +78,7 @@ done;
 
 echo "Deploying driver"
 sed -i'' "s,amazon/aws-efs-csi-driver,$IMAGE_NAME," deploy/kubernetes/overlays/dev/kustomization.yaml
```

Comment:
This script doesn't seem to be used anyway.

Reply:
You're right, I did a grep of latest and made changes everywhere. Technically someone could still use it (instead of using https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/tester/e2e-test-config.yaml which requires the k8stester binary), but probably we should remove it so we don't have to maintain it.